config/networks.json

```diff
@@ -1147,6 +1147,46 @@
     "deployedWithSolcVersion": "0.8.17",
     "create3Factory": "0x93FEC2C00BfE902F733B57c5a6CeeD7CD1384AE1"
   },
+  "tron": {
+    "name": "tron",
+    "chainId": 728126428,
+    "nativeAddress": "0x0000000000000000000000000000000000000000",
+    "nativeCurrency": "TRX",
+    "wrappedNativeAddress": "TNUC9Qb1rRpS5CbWLmNMxXBjyFoydXjWFR",
+    "status": "active",
+    "type": "mainnet",
+    "rpcUrl": "https://api.trongrid.io",
+    "verificationType": "tronscan",
+    "explorerUrl": "https://tronscan.org",
+    "explorerApiUrl": "https://apilist.tronscan.org/api",
+    "multicallAddress": "",
```

Comment:
I think this was added now and could be updated

Reply:
Fixed: 7595e503